caf-audit-elasticclient/src/main/java/com/hpe/caf/auditing/elastic/ElasticAuditConnection.java

```diff
@@ -26,36 +26,45 @@
 public class ElasticAuditConnection implements AuditConnection {
 
     private final TransportClient transportClient;
-    private ElasticAuditIndexManager indexManager;
+    private final ElasticAuditIndexManager indexManager;
```

Comment:
This is causing the build to fail, I think that you should remove the final.

Reply:
@davidegph Removed that final